src/append_only_data.rs

```diff
@@ -282,27 +296,26 @@ impl Permissions for PubPermissions {
 }
 
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
-pub struct Owners {
-    owners: PublicKeySet,
+pub struct Owner {
+    public_key: PublicKey,
     /// The current index of the data when this ownership change happened
     data_index: u64,
     /// The current index of the permissions when this ownership change happened
-    permission_entry_index: u64,
+    permissions_index: u64,
 }
 
 #[derive(Clone, Serialize, Deserialize, PartialEq, PartialOrd, Ord, Eq, Hash)]
 struct AppendOnly<P: Permissions> {
-    name: XorName,
-    tag: u64,
+    address: Address,
     data: Vec<(Vec<u8>, Vec<u8>)>,
     permissions: Vec<P>,
-    owners: Vec<Owners>,
+    owners: Vec<Owner>,
```

Comment:
Maybe add a comment here stating that this is a history of ownership rather than a list of owners? Vec< Owner > doesn’t convey that I think